api/napi/index.ts

```diff
@@ -9,7 +9,7 @@ export { Diagnostic, Window, Brush, Color, ImageData, Point, Size, SlintModelNot
  * set on dynamic Model<T> instances and can be used to notify the run-time
```

Comment:
While you're at it, could you change this to a link to Model so that it becomes clickable?

Reply:
done